### PR TITLE
[RIGA-2017] Initial commit

### PR DESCRIPTION
**Pull request by arthur-kovalenko**
[RIGA-2017] Initial commit for the DevOpsDays Riga conference, to set location and main info.

### CODE REVIEW

File: content/events/2017-riga/location.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{{< event_map >}}
+
+## Getting there and accommodation
+
+### Transportation
+
+### Parking spots
+Paid parking spots are available within few minute walk to the hotel.
+
+##### A note to out of towners in in regards to obtaining a reloadable card for public transportation.
```
Comment (phrawzty) [inline content/events/2017-riga/location.md]: This seems like a weird header - maybe it's supposed to be a section with some content instead?

File: content/events/2017-riga/propose.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+tags = ["riga","riga-2017"]
+type = "event"
+title = "Propose"
+heading = "DevOpsDays Riga 2017 - Propose a session"
++++
+  {{< cfp_dates >}}
+
+<hr>
+
+### Even if you don't propose, please consider {{< event_link page="proposals" text="commenting on proposals submitted by others" >}}
```
Comment (phrawzty) [inline content/events/2017-riga/propose.md]: You don't have a `proposals` page, so this link 404's. My suggestion would be to remove this line entirely - public commentary on proposals is a legacy artefact that most events don't bother using anyway.

File: content/events/2017-riga/sponsor.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  <tr>
+    <th class="tg-c7c7" rowspan="2">PACKAGES</th>
+    <th class="tg-mmvm">SILVER</th>
+    <th class="tg-mmvm">GOLD<br></th>
+    <th class="tg-mmvm">PLATINUM<br></th>
+  </tr>
+  <tr>
+    <td class="tg-y49x"></td>
+    <td class="tg-h7od">max 8</td>
+    <td class="tg-h7od">max 5</td>
```
Comment (phrawzty) [inline content/events/2017-riga/sponsor.md]: Given what you're promising in the Platinum package, five feels like a big number; my advice would be to reach out the DoD community for feedback here.

File: data/events/2017-riga.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+#  - name: workshop
+  - name: program
+  - name: propose
+#    url: https://<url>
+  - name: location
+  - name: registration
+#  - name: speakers
+  - name: sponsor
+  - name: contact
+#  - name: conduct
```
Comment (phrawzty) [inline data/events/2017-riga.yml]: All devopsdays events *must* have a Code of Conduct. Feel free to use the default CoC (included in this PR), or if you'd like to use your own, please append it to the PR. Either way this menu item will need to be uncommented before merging.

**Comment by phrawzty** [comment]
If you haven't already, please send the list of organisers (names & email addresses) to core@ so that we can create the mailing list and set up the Slack invites.

File: content/events/2017-riga/sponsor.md (hunk 3 above)
Comment (arthur-kovalenko) [inline content/events/2017-riga/sponsor.md]: Then again, the price is not that high, and we have seen from other conference, that there are many sponsors in the "gold/platinum" range.
Either way, made changes that you flagged, so hopefully, the PR could be approved :)

**Comment by phrawzty** [comment]
🇱🇻:+1:

**Pull request merged**